run_unit_tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+VERSION="dev"
+SHA=$(git rev-parse --verify HEAD)
+BUILDDATE=$(date '+%Y/%m/%d %H:%M:%S %Z')
+GOVERSION=$(go version)
+LDFLAGS="-X 'main.version=${VERSION}' -X 'main.sha=${SHA}' -X 'main.builddate=${BUILDDATE}' -X 'main.goversion=${GOVERSION}'"
+
+go test $(go list ./... | grep -v /vendor/) 
+
+echo "Unit tests: Done"
```

Comment:
This line scares me. We are saved by the `set -e` on top but if that's not there, this echo will cause the unit tests to break. It will make the script always exit with status 0 even if the tests fail.

I would prefer if we remove it in a new PR.